未安装 JustEnoughGuide 的情况下，无法使用 Pinyin 库

Plugin Networks vBuild 201 (git 770fc26) generated an exception while executing task 68080
java.lang.NoClassDefFoundError: Could not initialize class com.github.houbb.pinyin.util.PinyinHelper
        at NetworksExpansion-Build 201 (git 770fc26).jar//com.ytdd9527.networksexpansion.core.items.machines.AbstractGridNewStyle.lambda$getEntries0$4(AbstractGridNewStyle.java:434) ~[?:?]
        at java.base/java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:196) ~[?:?]
        at java.base/java.util.HashMap$EntrySpliterator.forEachRemaining(HashMap.java:1859) ~[?:?]
        at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:570) ~[?:?]
        at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:560) ~[?:?]
        at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:635) ~[?:?]
        at java.base/java.util.stream.AbstractPipeline.evaluateToArrayNode(AbstractPipeline.java:291) ~[?:?]
        at java.base/java.util.stream.ReferencePipeline.toArray(ReferencePipeline.java:652) ~[?:?]
        at java.base/java.util.stream.ReferencePipeline.toArray(ReferencePipeline.java:658) ~[?:?]
        at java.base/java.util.stream.ReferencePipeline.toList(ReferencePipeline.java:663) ~[?:?]
        at NetworksExpansion-Build 201 (git 770fc26).jar//com.ytdd9527.networksexpansion.core.items.machines.AbstractGridNewStyle.getEntries0(AbstractGridNewStyle.java:444) ~[?:?]
        at NetworksExpansion-Build 201 (git 770fc26).jar//com.ytdd9527.networksexpansion.core.items.machines.AbstractGridNewStyle.getEntries(AbstractGridNewStyle.java:385) ~[?:?]
        at NetworksExpansion-Build 201 (git 770fc26).jar//com.ytdd9527.networksexpansion.core.items.machines.AbstractGridNewStyle.lambda$updateDisplay$2(AbstractGridNewStyle.java:256) ~[?:?]
        at org.bukkit.craftbukkit.scheduler.CraftTask.run(CraftTask.java:78) ~[purpur-26.1.2.jar:26.1.2-2591-bb69e15]
        at org.bukkit.craftbukkit.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:57) ~[purpur-26.1.2.jar:26.1.2-2591-bb69e15]
        at com.destroystokyo.paper.ServerSchedulerReportingWrapper.run(ServerSchedulerReportingWrapper.java:22) ~[purpur-26.1.2.jar:?]
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1090) ~[?:?]
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:614) ~[?:?]
        at java.base/java.lang.Thread.run(Thread.java:1474) ~[?:?]
Caused by: java.lang.ExceptionInInitializerError: Exception java.lang.NoClassDefFoundError: com/github/houbb/heaven/support/instance/impl/Instances [in thread "Craft Scheduler Thread - 75 - Networks"]
        at NetworksExpansion-Build 201 (git 770fc26).jar//com.github.houbb.pinyin.support.segment.PinyinSegments.defaults(PinyinSegments.java:32) ~[?:?]
        at NetworksExpansion-Build 201 (git 770fc26).jar//com.github.houbb.pinyin.bs.PinyinBs.<init>(PinyinBs.java:34) ~[?:?]
        at NetworksExpansion-Build 201 (git 770fc26).jar//com.github.houbb.pinyin.bs.PinyinBs.newInstance(PinyinBs.java:90) ~[?:?]
        at NetworksExpansion-Build 201 (git 770fc26).jar//com.github.houbb.pinyin.util.PinyinHelper.<clinit>(PinyinHelper.java:32) ~[?:?]

--- a/src/main/java/io/github/sefiraat/networks/Networks.java
+++ b/src/main/java/io/github/sefiraat/networks/Networks.java
@@ -339,6 +339,22 @@ public class Networks extends JavaPlugin implements SlimefunAddon {
             .version("1.14.0")
             .build();
         libraryManager.loadLibrary(opencc4j);
+
+        getLogger().info("正在加载 heaven");
+        Library heaven = Library.builder()
+            .groupId("com{}github{}houbb")
+            .artifactId("heaven")
+            .version("0.13.0")
+            .build();
+        libraryManager.loadLibrary(heaven);
+
+        getLogger().info("正在加载 nlp-common");
+        Library nlp = Library.builder()
+            .groupId("com{}github{}houbb")
+            .artifactId("nlp-common")
+            .version("0.0.5")
+            .build();
+        libraryManager.loadLibrary(nlp);
     }
 
     public MinecraftVersion getMCVersion() {

